src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -90,9 +138,12 @@ const ChoiceType = PropTypes.shape({
 CustomFieldInputSingleChoice.propTypes = {
   id: PropTypes.string.isRequired,
   choices: PropTypes.arrayOf(ChoiceType),
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
I only included these because I could not figure out how to get lint to see that I was definitely using these props. If y'all have a way to get lint to chill out without these, that'd be great